third-party/jemalloc/Makefile

```diff
@@ -2,7 +2,7 @@ ifndef CHPL_MAKE_HOME
 export CHPL_MAKE_HOME=$(shell pwd)/../..
 endif
 
-CHPL_MAKE_HOST_TARGET = --target
+CHPL_MAKE_HOST_TARGET ?= --target
```

Comment:
I wonder if instead of having a default we should just error if this isn't set by the calling makefile

Reply:
Yeah that sounds good. Initially I think I made this change before specifying for the runtime build as well so it shouldn't be necessary at this point

Reply:
That would be ideal, but it looks like its already set with `ifndef` in `make/Makefile.base` so we can't easily error on it not being set. Introduce another jemalloc var that specifically has to be set? Leave as is?

Reply:
Leave as-is